app/components/certificate/certificate-available.tsx

```diff
@@ -28,14 +28,14 @@ export default function CertificateAvailable({
       <CertificateDisplay
         title="Public Key"
         value={publicKey}
-        description="Public: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+        description="Public key is intended to be shared widely and is used for encrypting data. Public key is used to encrypt data that can only be decrypted by the corresponding private key. In other words, anyone can use the public key to encrypt data, but only the owner of the private key can decrypt it."
```

Comment:
```suggestion
        description="Public key is intended to be shared widely and is used for encrypting data. The encrypted data can only be decrypted by the corresponding private key. In other words, anyone can use the public key to encrypt data, but only the private key can be used to decrypt it."
```
I think you can slightly modify this. This also clarifies that not only the owner, but anyone who has the private key can use it to decrypt the data